Automattic/jetpack
# Changelogger: Add New Prompt for User Change

## Proposed changes:

Per some recent discussions about how to properly format changelogs, this PR adds an additional prompt to changelogger to check if the change is user facing. If it's not, it automatically adds the `other` type so it won't appear in the readme.txt during release.

### Other information:


## Jetpack product discussion


pf4qpu-9d-p2
## Does this pull request change what data or activity we track or use?


no
## Testing instructions:





You can use `jetpack changelog add <plugin>` to test and make sure the new prompt appears. If you answer Yes, it should then ask you for the type of change after. If you answer no, it should skip it and automatically give it the `other` type.
